docs/interface.md

```diff
@@ -102,12 +102,10 @@ results = lm_eval.simple_evaluate( # call simple_evaluate
 )
 ```
 
-See https://github.com/EleutherAI/lm-evaluation-harness/blob/365fcda9b85bbb6e0572d91976b8daf409164500/lm_eval/evaluator.py#L35 for a full description of all arguments available. All keyword arguments to simple_evaluate share the same role as the command-line flags described previously.
+See https://github.com/EleutherAI/lm-evaluation-harness/blob/main/lm_eval/evaluator.py#:~:text=simple_evaluate `simple_evaluate()` for a full description of all arguments available. All keyword arguments to simple_evaluate share the same role as the command-line flags described previously.
```

Comment:
You may also try relative path, somethng like 
```
[`simple_evaluate()`](lm_eval/evaluator.py#:~:text=simple_evaluate)
```
in case same doc will be viewed from cloned repo or forked branch/commit and follow that state of code